packages/auth/lib/index.d.ts

```diff
@@ -472,7 +472,21 @@ export namespace FirebaseAuthTypes {
   /**
    * Contains information about a second factor.
    */
-  export interface MultiFactorInfo {
+  export type MultiFactorInfo = PhoneMultiFactorInfo | TotpMultiFactorInfo;
+
+  export interface PhoneMultiFactorInfo extends MultiFactorInfoCommon {
+    factorId: 'phone';
+    /**
+     * The phone number used for this factor.
+     */
+    phoneNumber: string;
+  }
+
+  export interface TotpMultiFactorInfo extends MultiFactorInfoCommon {
```

Comment:
There's no additional properties on this yet, but felt it is still useful to represent it for future / consistency with the underlying SDK's